### PR TITLE
creature_template - added missing fields

### PR DESCRIPTION
**Pull request by Shard-MW**
## Description
Updating creature_template : missing some fields, and "deleting" non-existants fields.

## Related Issue
Some fields were missing from the creature_template doc.
I noticed it while working on creatures. AC Rev : 1f4eefe1bea6d995fa110169afe25ac5c61b0254

## Motivation and Context
Required to improve the wiki.

## How Has This Been Tested?
Compared the wiki and the creature_template table.
Tested by creating new creatures in game.

### CODE REVIEW

File: docs/creature_template.md [hunk 1]
```diff
@@ -27,11 +27,14 @@ This table contains the description of creatures. Each spawned creature is an in
...
+| [attackpower](#creature_template-attackpower)                     | int(10) unsigned      | NO   |     | 0       |       |                                      |
 | [BaseAttackTime](#creature_template-BaseAttackTime)               | int(10) unsigned      | NO   |     | 0       |       |                                      |
 | [RangeAttackTime](#creature_template-RangeAttackTime)             | int(10) unsigned      | NO   |     | 0       |       |                                      |
-| [BaseVariance](#creature_template-BaseVariance)                   | float                 | NO   |     | 1       |       |                                      |
-| [RangeVariance](#creature_template-RangeVariance)                 | float                 | NO   |     | 1       |       |                                      |
+| BaseVariance                                                      | float                 | NO   |     | 1       |       |                                      |
+| RangeVariance                                                     | float                 | NO   |     | 1       |       |                                      |
```
Comment (FrancescoBorzi) [inline docs/creature_template.md]: there are no such fields in AC, so we should delete them completely

**Comment by FrancescoBorzi** [comment]
good job @Shard-MW , please check my comment

File: docs/creature_template.md (hunk 1 above)
Comment (Shard-MW) [reply docs/creature_template.md]: Alright, didn't wanted to delete them completely, but if you ask for I can do it hahaha

**Comment by BarbzYHOOL** [comment]
nice

**Pull request merged**